### ISSUE TITLE
about the label and loss

### ISSUE THREAD

**Issue by nmvbxcz** [NVlabs/ssn_superpixels#34]
the article said the R can be semantic label (as one-hot encoding) or optical flow maps
but the whole Algorithm 1 has no R
i dont know what is the data X and what is the label Y for backpropagation in the network, and the loss ||R-R*|| in the article seems like autoencoder, which dosen't need any label.
does the R is I or F in the Algorithm 1?

**Comment by varunjampani** [comment]
SSN takes image (I) as input and produces superpixels. 'R' is only used in loss function. That's why there is no R in the algorithm 1. Network produces Q and we use that Q to map R to superpixels and then back to pixels (see sec. 4.3 and eq. 5).

**Comment by nmvbxcz** [comment]
> SSN takes image (I) as input and produces superpixels. 'R' is only used in loss function. That's why there is no R in the algorithm 1. Network produces Q and we use that Q to map R to superpixels and then back to pixels (see sec. 4.3 and eq. 5).

thanks，i almost understand the design of this loss.
